src/widget/wtrackmenu.cpp

```diff
@@ -47,6 +47,11 @@ WTrackMenu::WTrackMenu(QWidget* parent,
     m_pNumPreviewDecks = new ControlProxy(
             "[Master]", "num_preview_decks", this);
 
+    if (!trackModel) {
+        // Warn if any of the chosen flags depend on a TrackModel
+        VERIFY_OR_DEBUG_ASSERT((m_eTrackModelFilters & flags) == 0) {
```

Comment:
The fallback here should be to remove any unsupported flags from `m_eFilters`.

Reply:
Oh true. This should be added. Thanks.